app/components/wallet/send/WalletSendForm.scss

```diff
@@ -7,6 +7,10 @@
   padding-bottom: 20px;
   padding-left: 20px;
 
+  .warningBox {
+    margin-bottom: 30px;
```

Comment:
maybe irrelevant but I've seen 24px everywhere except here

Reply:
cc @Marta-Ryzhok

Reply:
@sarveshwar-singh @v-almonacid to where these paddings are related?

Reply:
Actually it's just in WalletSendConfirmationDialog where I saw margin of 24px.

@Marta-Ryzhok it is the margin between the warning message and the component below (which is the receiver input address in this case)